Assignment2/test_server.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from server import webhook_message, app
+
+@pytest.fixture
+def client():
+    with app.test_client() as client:
+        yield client
+
+def test_webhook_message(client):
```

Comment:
I think a description of the test case would be good, including the expected result :)